types/angular-desktop-notification/tslint.json

```diff
@@ -0,0 +1,7 @@
+{ 
+    "extends": "dtslint/dt.json" ,
+    "rules": {
+        "interface-name": false,
+        "max-line-length": false
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`, you can disable them in code(`// tslint:disable-next-line max-line-length`) if you have to.

Reply:
@plantain-00 Thanks, I used `disable-next-line` above interface definition